bdc_scripts/radcor/sentinel/correction.py

```diff
@@ -0,0 +1,78 @@
+# Python Native
+import logging
+import os
+import re
+import shutil
+import time
+from json import loads as json_parser
+
+# 3rdparty
+from requests import get as resource_get
+
+# BDC Scripts
+from bdc_scripts.config import Config
+
+
+def search_recent_sen2cor280(safeL2Afull):
+    safe = safeL2Afull.replace( os.path.basename(safeL2Afull).split('_')[3], 'N9999')
+    safe_pattern = '_'.join( os.path.basename(safe).split('_')[0:-1])
+    dirname = os.path.dirname(safeL2Afull)
+    dirs_L2 = [os.path.join(dirname,d) for d in os.listdir(dirname) if re.match(safe_pattern, d)]
+    return dirs_L2
+
+def correction_sen2cor255(self, scene):
```

Comment:
What is `self`?

Move the `SentinelTask.sen2cor_done` to `correction.py` file. So you don't need to pass task reference.